go.mod

```diff
@@ -6,15 +6,23 @@ module github.com/hyperledger/aries-framework-go
 
 require (
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
-	github.com/go-openapi/runtime v0.19.3
```

Comment:
How will you generate the openapi/swagger docs?

Reply:
We just have to run command like swagger generate spec -b cmd/aries-agentd/main.go -o ./swagger.yml to generate spec. Swagger(OpenAPI) is going to read annotations and generate spec.
I am adding that command as validation while doing #94 before running any test.
Since we are using annotations here, we are not going to having direct dependencies on open API libraries.